scheduler/cloudformation/cloudformation.go

```diff
@@ -167,6 +167,17 @@ type SubmitOptions struct {
 	// When true, does not make any changes to DNS. This is only used when
 	// migrating to this scheduler
 	NoDNS bool
+
+	// When provided, will generate a new RestartKey for the target.
+	RestartTarget string
+}
+
+func (s *Scheduler) Restart(ctx context.Context, app *scheduler.App, p string) error {
```

Comment:
Also, to keep things simple to start, and because of the parameter limit on cloudformation, we may wanna just start with a `Restart(context.Context, *scheduler.App)` method and then add process restarts later.